packages/slate-editor/src/components/EditorBlock/EditorBlock.module.scss

```diff
@@ -70,6 +70,33 @@ $editor-block-outline-width: 6px;
     }
 }
 
+.LoadingIndicator {
+    display: block;
+    position: absolute;
+    left: 0;
+    top: 0;
```

Comment:
_Comment in an unrelated place_

Here's a potential bug:


https://github.com/prezly/slate/assets/67554982/f8f9b429-fcf4-4439-8328-8c9b0f504b97


Steps to reproduce:

1. Add a gallery
2. Add some images
3. Click on the "Edit"
4. Close the modal
5. It gets stuck in the loading state

But it shouldn't.

Reply:
Great catch! Thanks!

Reply:
dbe8783fadb469eaf6925630a56cfe4a99a98f85